### PR TITLE
No legacy platform variables

### PR DESCRIPTION
**Pull request by swansontec**
### CHANGELOG

 none

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [x] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204256160179278

### CODE REVIEW

File: src/components/modals/FlipInputModal.tsx [hunk 1]
```diff
@@ -251,9 +249,11 @@ export class FlipInputModalComponent extends React.PureComponent<Props, State> {
   }
 }
 
+const deviceHeight = Dimensions.get('window').height
+
 const getStyles = cacheStyles((theme: Theme) => ({
   hackContainer: {
-    flex: deviceHeight <= MINIMUM_DEVICE_HEIGHT ? 1 : 0
+    flex: deviceHeight <= 580 ? 1 : 0
```
Comment (samholmes) [inline src/components/modals/FlipInputModal.tsx:256]: Why was the constant removed? I like that it described the value. Can we at least move it to the top of this file?

Comment (samholmes) [inline src/components/modals/FlipInputModal.tsx:256]: Why was the constant removed? I like that it described the value. Can we at least move it to the top of this file?

File: src/components/modals/HelpModal.tsx [hunk 2]
```diff
@@ -162,8 +163,8 @@ const getStyles = cacheStyles((theme: Theme) => ({
     height: theme.rem(2.25)
   },
   footer: {
-    marginTop: PLATFORM.deviceHeight < theme.rem(42) ? 0 : theme.rem(1.5),
-    paddingVertical: PLATFORM.deviceHeight < theme.rem(42) ? theme.rem(0.25) : theme.rem(0.5),
+    marginTop: deviceHeight < theme.rem(42) ? 0 : theme.rem(1.5),
+    paddingVertical: deviceHeight < theme.rem(42) ? theme.rem(0.25) : theme.rem(0.5),
```
Comment (samholmes) [inline src/components/modals/HelpModal.tsx:167]: Might be an opportunity to explain this logic.

**Review by samholmes — APPROVED** [review]
Approved with minor feedback

File: src/components/modals/FlipInputModal.tsx (hunk 1 above)
Comment (swansontec) [reply src/components/modals/FlipInputModal.tsx:256]: The constant did not describe anything. If I gave you a constant named "MINIMUM_DEVICE_HEIGHT" and no context, what would you guess it meant? Probably "the app won't run on phones less than 580px tall", or something incorrect like that.

The CSS section is always full of numbers. That's what they are for - to describe how things look. Given it's new home in the CSS section, the number is no longer misleading. It's perfectly obvious that the modal flexes only if the phone is below 580px in size now.

File: src/components/modals/HelpModal.tsx (hunk 2 above)
Comment (swansontec) [reply src/components/modals/HelpModal.tsx:167]: I have no idea.

File: src/components/modals/FlipInputModal.tsx (hunk 1 above)
Comment (samholmes) [reply src/components/modals/FlipInputModal.tsx:256]: Fair enough.

File: src/components/modals/HelpModal.tsx (hunk 2 above)
Comment (samholmes) [reply src/components/modals/HelpModal.tsx:167]: Ha! Okay

**Pull request merged**